### DISCUSSION TITLE
[2024 project] apply_product team

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#202] [iai3-o4s]
Bold from below collaboration plan, in parallel with #201 

The MIT PI Charles Fine and coPI Abdullah Almaatouq have built two parallel teams to combine theory and practice for this project. The theory team (Abdullah, Angie) will have biweekly meetings to enhance technical feasibility of the GPS such as developing computational statistical models that decompose effects of team, process, task, industry from the log of past startups. These models can help test generalizability of existing theories. Further experiments can be designed using integrative experiment design which contributes to generate cumulative empirical and theoretical knowledge (Almaatouq et al (2024)). Concurrently, **the practice team (Charlie, Doug, Angie, Anup) will have biweekly meetings to enhance operational feasibility. This involves translating statistical evidence from the theory team into actionable business insights and continuously integrating insights into custom tools, and discussing the architecture for better collaboration with Google to use its data and computational resources. MVP implemented on MIT digital entrepreneurship platform and its enthusiastic reception from our beta users (5,000) shows potential of our product. Based on this pipeline, GPS can be delivered to startups to help them craft business plans in a fraction of the traditional time.**

**Comment by hyunjimoon** [comment]
### JB, Angie's meeting:
- Observation 1: Go deeper in the pivot / reconfigurations in the evolution of a startup or research line by transferring knowledge from exaptation research
- Observation 2: Examine Bob Horn complex visual representation (mega mess mapping)
- Mockup 1: From this qualitative pre-study, we can then create a mockup of an explainable evolutionnary map/tool/platform fro startup innovation in biotech and semic
- take inspiration to make it from stat, business, simulation (SBS)
- clock and compass

### JB, Charlie, Angie's meeting:
based on pre-alinged JB, Angie's action item, participants discussed the importance of focusing on concrete, actionable steps in startups, adapting to unexpected circumstances, and incorporating situated contingencies into the planning process. They emphasized the need for a systematic approach to making progress, collecting data, and understanding the evolution of business models. Additionally, they highlighted the potential for developing a tool to assist with biotech product management and the challenges of using GPT products versus Orbit. 

### Action Items
#### Goal
- [x] define goal and ask charlie and jb's verification: deliver AI tool that supports startup growth with frontend (animation that maps startup action with outcomes; [mockup](https://probsys.github.io/AutoGP.jl/stable/)) and backend SBS ((statistics, business, simulation) theories ) + transfer knowledge from expatiation research to startup business theory

####  Asset
- [x]  Schedule a monthly Zoom call for further discussion and ideation.
- [ ] log "gantt chart dynamics" i.e. chain reaction of NASA's planning tools w.r.t. mission change in https://github.com/Data4DM/BayesSD/discussions/203 
- [ ] Look into potential funding sources like [Google venture](https://www.gv.com/) and biotech VCs.

#### Technology
- [ ] build examples of what biotech and semiconductor could learn from each other in #190 (examples of potential Spandrel between two industries with different clockspeed and physicalness including physics and biology, science and technology as in https://github.com/Data4DM/BayesSD/discussions/169#discussion-5757329)
- [ ] learn Bob Horn's work on information mapping and visual representation of complexity, by attending Thursday talk
- [ ] find upto two UROPs  https://urop.mit.edu/mentors/ who can help design and code database: this requires 1) how to encode startup evolution (e.g. should I set each action A, AG, AGC, AGCT as separate row?) 2) startup action taxonomy (define the right abstraction level)

#### Customer
- [x]  Reach out to biotech experts like Natalie Kuldell to learn more on biotech growth and tools they need + visit biotech incubators

#### Recurring
- [x] Make 1-month plan and share status two days before monthly meetings (share biweekly progress mail)

**Reply by hyunjimoon** [reply]
detailed summary of meeting (dictated by otter.ai with 80% accuracy)

Planning and adapting to unexpected events in a startup.
Speaker 3 discusses the challenges of planning a space mission, including the need to consider multiple stakeholders and unexpected contingencies.
Speaker 1 compares the complexity of planning a space mission to developing a production line for a car manufacturer, highlighting the differences in predictability and adaptability.
Speaker 1 discusses their experience in the startup industry, mentioning the importance of understanding the situated contingencies and unexpectedness in planning and design.
Speaker 1's PhD research focused on end-user programming and critical configurations, drawing inspiration from project Athena in the 80s at MIT.
Using visual representation to explain complex systems.
Speaker 1 discusses the concept of "spandrels" in architecture, highlighting how the creation of arches leads to morphological byproducts like empty spaces that can be decorated and studied.
Speaker 1 also explores the origin of email, citing how engineers observed people sending zero-kilobyte files with empty contents but meaningful titles, leading to the creation of the email protocol.
Speaker 1 discusses the importance of visual representation in explaining complex systems, citing the example of Bob Langer's work at MIT.
Speaker 1 plans to contact Bob Langer and NASA's JPL to collaborate on a tool for harnessing three-dimensional statistics and simulations to create a mock-up for business relations and simulations.
Startup vision, strategy, and execution.
Speaker 2 expresses excitement about research with JB, desires concrete action points.
Speaker 4: Pivoting means changing strategy while maintaining mission (e.g., going to the moon).
Speaker 2: Probing and reconfiguring lower-level actions to align with changed vision (e.g., from biotech to semiconductor).
Interdisciplinary approaches in technology and biology.
Speaker 3 compares clock speed in technology industries, suggesting that faster-evolving industries like biotech may have lessons for slower-growing ones.
Speaker 3 hypothesizes that slower industries could learn from faster ones, but is unsure if clock speed is the right metric for comparison.
Ed Boyden, a professor at MIT, invented optogenetics, a field that combines physics and biotech.
Vaughn, an organizer of the talk on Thursday, is bringing together people from different fields to discuss the intersection of physics and biotech.
Interdisciplinary collaboration in tech and science.
Speaker 1 mentions Bob, a design thinking expert, and their work on sustainable design, including a tool to help designers visualize the impact of their decisions.
Speaker 1 also mentions Grunfeld, a citizen intrapreneur and scientist studying startups and innovation, and their potential interest in collaborating on a project.
Speakers discuss potential for interdisciplinary collaboration between biology and computer science fields, with one speaker expressing interest in "parasitizing" the biology field for ideas.
Innovation, AI, and entrepreneurship in Israel and France.
French ambassador in Israel discusses innovation and diplomacy with Israeli entrepreneurs.
Speaker 1 mentions NASA's design process and the importance of involving multiple stakeholders, including engineers and UX designers.
Speaker 6 discusses intermediate deliverables and roles in the design process.
Biotech innovation and entrepreneurship.
Speaker 6 aims to be on the moon in 5 years, while Speaker 1 focuses on simulating moon missions on Earth.
Speaker 1 highlights the importance of being embedded in the field of biotech, connecting with people in the industry, and learning from their perspectives.
Startup development and evolution.
Developing a systematic approach to organize and analyze data on startups, including creating a taxonomy for categorizing different types of startups and their actions.
Speakers discuss the importance of recording and analyzing actions and learning notes in the context of business model development.
Speaker 1 discusses the evolution of a company from a startup to a larger organization, highlighting the challenges of scaling up and institutionalizing processes.
Speaker 1 uses the Business Model Canvas as a tool to visualize and analyze the evolution of the business model over time.
Mapping actions to outcomes in a startup.
Speaker 2 is working on a project to map the sequence of actions and decisions made by a company, with the goal of creating a compass to show the mechanism of success.
Speaker 1 is interested in using the project as a facilitation assistant to write a running Pro, and the group discusses the potential for transferring the project's hyper parameter pooling to different industries.
Using AI tools for product management.
Speaker 3 brainstorms product ideas for urban car owners with stolen keys.
Speaker 3 impressed with AI's ability to provide creative solutions to product management problems, but also recognized the limitations of AI in deconstructing and explaining its answers.
Speaker 1 shared a story about students using GPT to write essays, and the importance of critical thinking and research to evaluate the quality of AI-generated content.
AI-powered tool for biotech companies.
Speaker 1 suggests developing a tool to help with biotech company selection and design, with the goal of focusing on a single tool to grow and improve.
Speaker 2 discusses the potential for a product management app to help with Orbot development, with the ability to update the PID and improve the product.
Speaker 1 discusses the importance of documentation in their research, particularly in hand-hand practice, and offers to schedule a monthly Zoom meeting to discuss further.
Speaker 2 shares their experience of losing a translation project in Kibera and suggests scheduling a future meeting to discuss logistics.
Speaker 1 mentions Google's venture arm and how it could be interested in funding research projects (57:06).
Speaker 2 discusses the potential for partnerships with VCs and the importance of interpersonal connections for fundraising (58:34).
Sustainable construction materials and benchmarking.
Speaker 1 and 2 discuss meeting someone next week and taking a picture.
Researchers develop sustainable building materials using waste materials and electronics to reduce carbon footprint.
Speaker 2 seeks clarification on benchmark definition during meeting.

**Reply by jeanbaptiste** [reply]
Regarding NASA, look at [TaskNet](https://ai.jpl.nasa.gov/public/projects/tasknet/) and [MEXEC](https://ai.jpl.nasa.gov/public/projects/mexec/). Look particularly at the Biotechnology scenario that Tiago (with whom we submitted a grant application between JPL and MIT) developed (at the end of the second paper page 412.

<img width="918" alt="image" src="https://github.com/Data4DM/BayesSD/assets/90195/80e7e014-5ff1-422b-8252-f02aa377be9a">

[IntEx-2020-MEXEC-2.pdf](https://github.com/Data4DM/BayesSD/files/14643442/IntEx-2020-MEXEC-2.pdf)

<img width="1112" alt="image" src="https://github.com/Data4DM/BayesSD/assets/90195/68d7017a-4e84-4993-adf4-658996388519">

[vaquero-et-al-2023-property-based-brittleness-analysis-of-temporal-networks.pdf](https://github.com/Data4DM/BayesSD/files/14643464/vaquero-et-al-2023-property-based-brittleness-analysis-of-temporal-networks.pdf)

**Reply by jeanbaptiste** [reply]
<img width="1090" alt="image" src="https://github.com/Data4DM/BayesSD/assets/90195/9f2514da-ad14-4c10-b3cf-6d713b02ed15">

**Reply by hyunjimoon** [reply]
I love how you're sharing images, which have the highest information concentration in the paper, proxied by authors' time investment. Comparing images on their structural level is exactly the mechanism behind my reconfiguration!

**Reply by hyunjimoon** [reply]
@jeanbaptiste re: system complexity, there has been attempts to connect computer complexity theory with system complexity by Jeannette Wing, as on of the five deep questions in computing

-  **(How) can we build complex systems simply?**
<img width="825" alt="image" src="https://github.com/Data4DM/BayesSD/assets/30194633/ec447617-6494-49eb-a173-634162dcfa76">

the other four are:  P = NP?,  What is computable? , What is intelligence? ,  What is information?

**Reply by hyunjimoon** [reply]
- entrepre. success
![image](https://github.com/Data4DM/BayesSD/assets/30194633/373b5efd-c30d-47a8-91a2-719cf02694e8)
[FreseGielnikPsychologyofentrepreneurshipannurev-orgpsych-2014.pdf](https://github.com/Data4DM/BayesSD/files/14696841/FreseGielnikPsychologyofentrepreneurshipannurev-orgpsych-2014.pdf)

**Comment by jeanbaptiste** [comment]
UROP info page https://urop.mit.edu/mentors/

**Reply by hyunjimoon** [reply]
job posted!
